R/RunDiagnostics.R

```diff
@@ -147,28 +174,12 @@ runCohortDiagnostics <- function(packageName = NULL,
     takepackageDependencySnapshot() %>%
     RJSONIO::toJSON(digits = 23, pretty = TRUE)
 
-
   exportFolder <- normalizePath(exportFolder, mustWork = FALSE)
   incrementalFolder <- normalizePath(incrementalFolder, mustWork = FALSE)
 
-  if (!is.null(inclusionStatisticsFolder)) {
```

Comment:
by removing inclusionStatisticsFolder - we are asserting that
- inclusion rules table always have to be on remote db and should not have been dropped prior to starting cohort diagnostics
- i think we should make this explicit and clear in some documentation

Reply:
The user doesn't need to do anything at all; CohortDiagnostics now uses CohortGenerator to populate and export the results. I don't know what the document should say? Don't delete the cohort tables until you've finished the analysis? This seems somewhat obvious.

Reply:
i think u r right

and if they have dropped the inclusion rules table from the server - they would not get inclusion rule statistics diagnostics.

Reply:
In terms of documentation -- if some folks dont see inclusion rule diagnostics, when one was expected - the obvious first question should be - did u drop the inclusion rule tables before running cohort diagnostics